tasks/install-from-source.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Clone Certbot into configured directory.
+  git:
+    repo: "{{ certbot_repo }}"
+    dest: "{{ certbot_dir }}"
+    version: "{{ certbot_version }}"
+    update: "{{ certbot_keep_updated }}"
+    force: yes
```

Comment:
I just copied it over from the current implementation of this role. If such an option is wanted, one could introduce it but this seems to be out of scope for this PR.